src/platforms/node/common/configuration/integrations/default-integrations.mdx

```diff
@@ -140,6 +125,25 @@ _Import name: `Sentry.Integrations.Modules`_
 
 This integration fetches names of all currently installed Node modules and attaches the list to the event. Once fetched, Sentry will cache the list for later reuse.
 
+## Modifying System Integrations
```

Comment:
This should go below `RequestData`, I think!

Reply:
Yes, that makes sense! And thanks for updating!